src/statsig/constants.ts

```diff
@@ -1,6 +1,35 @@
-import { StatsigDynamicConfigs, StatsigExperiments, StatsigParameter } from 'src/statsig/types'
+import {
+  StatsigDynamicConfigs,
+  StatsigExperiments,
+  StatsigFeatureGates,
+  StatsigParameter,
+} from 'src/statsig/types'
 import { NetworkId } from 'src/transactions/types'
 
+export const FeatureGates = {
+  [StatsigFeatureGates.SHOW_POSITIONS]: true,
+  [StatsigFeatureGates.SHOW_CLAIM_SHORTCUTS]: true,
+  [StatsigFeatureGates.ALLOW_HOOKS_PREVIEW]: true,
+  [StatsigFeatureGates.APP_REVIEW]: false,
+  [StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW]: false,
```

Comment:
Should we enable it? Looks like an interesting feature to be enabled by default.
```suggestion
  [StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW]: true,
```

Reply:
oh yes i went back and forth on this, i went with false because i think beefy and ckash (the only examples so far) don't need it but i think enabling it is also good